network/src/commonMain/kotlin/com/wire/kalium/network/api/user/logout/LogoutApi.kt

```diff
@@ -18,9 +21,25 @@ class LogoutImpl(private val httpClient: HttpClient, private val refreshToken: S
         }
     }
 
+    override suspend fun removeCookiesByIds(removeCookiesByIdsRequest: RemoveCookiesByIdsRequest): NetworkResponse<Unit> =
+        wrapKaliumResponse {
+            httpClient.post("$PATH_COOKIES/$PATH_REMOVE") {
+                setBody(removeCookiesByIdsRequest)
+            }
+        }
+
+    override suspend fun removeCookiesByLabels(removeCookiesWithIdsRequest: RemoveCookiesByLabels): NetworkResponse<Unit> =
+        wrapKaliumResponse {
+            httpClient.post("$PATH_COOKIES/$PATH_REMOVE") {
+                setBody(removeCookiesWithIdsRequest)
+            }
+        }
```

Comment:
I think the name of `removeCookiesByIdsRequest` should be `removeCookiesByLabelsRequest`, am I right? 
```suggestion
    override suspend fun removeCookiesByLabels(removeCookiesByLabelsRequest: RemoveCookiesByLabels): NetworkResponse<Unit> =
        wrapKaliumResponse {
            httpClient.post("$PATH_COOKIES/$PATH_REMOVE") {
                setBody(removeCookiesByLabelsRequest)
            }
        }
```